### PR TITLE
Finished Welcome Page Ticket

### PR DESCRIPTION
**Pull request by WillMash1**
Please review and approve pull request.

### CODE REVIEW

File: and-zero-frontend/src/App.css [hunk 1]
```diff
@@ -1 +1,11 @@
 
+
```
Comment (ANDSunya) [inline and-zero-frontend/src/App.css]: Delete blank lines above please

File: and-zero-frontend/src/pages/WelcomePage.js [hunk 2]
```diff
@@ -3,9 +3,21 @@
 function WelcomePage() {
     return (
         <>
-            <h1 class="text-3xl font-bold underline">
-                Hello world!
-            </h1>
+            <div className="hero min-h-screen" style={{ backgroundImage: `url("https://images.unsplash.com/photo-1542601906990-b4d3fb778b09?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=2813&q=80")` }}>
+            <div className="hero-overlay bg-opacity-60"></div>
```
Comment (ANDSunya) [inline and-zero-frontend/src/pages/WelcomePage.js]: Inconsistent indentation, please make sure they are consistent (e.g. elements within another element are indented by 1 tab)

File: and-zero-frontend/src/pages/WelcomePage.js [hunk 3]
```diff
@@ -3,9 +3,21 @@
...
+            <div className="hero-content text-center text-neutral-content">
+                <div className="max-w-md">
+                <h1 className="mb-5 text-5xl font-bold">Hello there</h1>
+                <p className="mb-5">Provident cupiditate voluptatem et in. Quaerat fugiat ut assumenda excepturi exercitationem quasi. In deleniti eaque aut repudiandae et a id nisi.</p>
+                <div className="flex flex-col">
+
```
Comment (ANDSunya) [inline and-zero-frontend/src/pages/WelcomePage.js]: Delete blank lines

**Review by ANDSunya — CHANGES_REQUESTED** [review]
We have added some comments for you to address

**Comment by WillMash1** [comment]
The suggested changes were made

**Pull request merged**